iOS/Flows/Conversation/Conversation Header/MembersViewController.swift

```diff
@@ -74,6 +76,16 @@ class MembersViewController: DiffableCollectionViewController<MembersCollectionV
             }
         }).store(in: &self.cancellables)
     }
+    
+    override func handleDataBeingLoaded() {
+        super.handleDataBeingLoaded()
+        
+        /// This is needed because sometimes the author get set before the datasource
```

Comment:
Triple slash should be used for documentation. This is an internal comment so double slash should be used.